cmd/cosign/cli/copy/copy.go

```diff
@@ -161,3 +169,10 @@ func remoteCopy(ctx context.Context, pusher *remote.Pusher, src, dest name.Refer
 	fmt.Fprintf(os.Stderr, "Copying %s to %s...\n", src, dest)
 	return pusher.Push(ctx, dest, got)
 }
+
+func normalize(h v1.Hash, prefix string, suffix string) string {
```

Comment:
I think we can get rid of this whole function, right? It's only ever called with `suffix` set to `""`.

Reply:
Yes, true, initially I thought in future, it might be possible (like in some other  cases I saw in code base) to add a suffix too but for now we probably don't need it, I will remove it and push it, thanks for the review!

Reply:
Done, thanks :))